config-connector/solutions/iam/kpt/member-iam/README.md

```diff
@@ -0,0 +1,84 @@
+Member IAM
+==================================================
+
+# NAME
+
+  member-iam
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to create a service account in your desired project, and grant it a specific role (defaults to `compute.networkAdmin`) in the project.
```

Comment:
Is it important to call out what the default role is here? I'd assume that the package consumer can already see it in YAML/setter form, and doesn't need to be told.

Reply:
Yes and no. After reading your comment, I rethought about it. I think calling out the default role is useful but just calling out the default role is not sufficient.

Why useful - Customers will probably read READMEs at the first trial, but they will unlikely read YAMLs. If compute.networkAdmin is a sensitive role to them, I'd prefer them know it sooner. 

Why not sufficient - I think what's missing in the doc is the output of the get commands (like list-setters), because customers won't know what is the right output if they've never used this package/kpt tool before. Solutions are for customers to get onboard, so we should expect customers are not sophisticated enough to figure out information they want by themselves. Providing the expected output for customers to compare with will ensure they are on the right track. It's just like how TF CFT READMEs do ([example](https://github.com/terraform-google-modules/terraform-google-iam/tree/master/modules/folders_iam)).

I'm in general a fan of providing redundant information to customers to avoid them missing important info. Even if it means harder maintenance. We all like TF documentation, and one reason I find it's useful is that they provide sufficient information even if sometimes it is redundant. E.g. for the default field in the resources, it is defined in the schema (go code), specified in the doc (maintained separately), and is sometimes also mentioned in the samples.

Reply:
As we discussed offline, let's just leave these kind details at the comfort level of the solution dev and update them later if we receive any customer feedback.

In regard to whether we are going to list all the setters in the README or not, we'll decide later.